### DISCUSSION TITLE
글자 추출 오류 시 개인이 수정할 수 있게 할 것인가

### DISCUSSION THREAD

**Discussion by wlsl10** [MajorIn-ABO/MajorIn_FE#21] [General]
글자 추출 하는 부분에서 잘 실행이 안 되어서 오타가 났을 때 
피그마에서는 다시 학생증을 업로드하라고 되어 있는데
![정보입력](https://github.com/MajorIn-ABO/MajorIn_FE/assets/77524688/c3e2c836-aa01-4941-94b1-0d33cbc409e3)
그렇게 했을 때도 계속 오류가 나는 경우를 생각해봐야 될 거 같은데

이때 사용자가 자신의 정보를 수정하게 할 수 있느냐는 고민해봐야 될 것 같아. 다들 의견 남겨줘
나는 이때 신분증 추출의 목적에 대해 생각을 해봐야 될 것 같은데
1. 편리함 2. 인증
이렇게 두 가지 일 것 같은데 두 가지 모두 고려해봤을 때
사용자가 수정할 수 있게 한 다음 바로 사용을 못 하고 우리가 인증을 하면 넘어가는 식으로 하면 될 것 같아!

**Comment by JangGusWjd** [comment]
근데 사용자가 수정을 하면 아예 처음부터 입력받는거랑 다를게 없어서 별도로 관리자한테서 인증할 수 있도록 해야할듯!
